### PR TITLE
add number of comments

### PR DESCRIPTION
**Pull request by tianlangwu**
This RP is for [issue-57](https://github.com/trybick/issue-collab/issues/57). The number of comments shows on the bottom left of each issue.
SS02

When I am run the command `npm run lint:fix` to check my formatting, it automatically formats something in the other files.

Let me know if I did anything incorrectly, thanks for the consideration.

### CODE REVIEW

File: src/components/search/SearchResultsContainer.jsx [hunk 1]
```diff
@@ -21,7 +21,7 @@ const SearchResultsContainer = ({ currentPage, onPageChange, results }) => {
   const formattedResults =
     results.items[0] &&
     results.items.map(item => {
-      const htmlUrl = item.html_url.split('/');
+      const { htmlUrl, comments } = [item.html_url.split('/'), item.comments];
```
Comment (trybick) [inline src/components/search/SearchResultsContainer.jsx]: Would it be possible to do this on a separate line than the htmlUrl - 
`const { comments } = item;`

File: src/components/search/SearchResult.jsx [hunk 2]
```diff
@@ -38,7 +39,8 @@ export const SearchResult = ({
         <p className="body-text">{bodyText}</p>
 
         <div className="metadata">
-          <div>{`${userName}/${repoName}`}</div>
+          <div>{`${userName}/${repoName}`} </div>
+          <div className="comments">{`Total comments ${comments}`}</div>
```
Comment (trybick) [inline src/components/search/SearchResult.jsx]: Could we do - 

`Comments: ${comments}`

**Review by trybick — COMMENTED** [review]
It looks really good 👍  However I am getting this error when I click the 'Get Issues' in the deploy preview, can you take a look? This might be fixed by resolving the comment about this line.

Screen Shot 2020-10-17 at 3 28 46 PM

**Comment by trybick** [comment]
Also sorry, there are some merge conflicts which were introduced by this PR  (if it helps to look at it 
 https://github.com/trybick/issue-collab/pull/60)

File: src/components/search/SearchResult.jsx (hunk 2 above)
Comment (tianlangwu) [reply src/components/search/SearchResult.jsx]: Sure, thanks for the advice.

File: src/components/search/SearchResultsContainer.jsx (hunk 1 above)
Comment (tianlangwu) [reply src/components/search/SearchResultsContainer.jsx]: I used to do it this way, 
ss03

but when I run the formating command, it said 
ss04

Comment (trybick) [reply src/components/search/SearchResultsContainer.jsx]: Oh gotcha. Please skip the lint:fix command for now

**Comment by tianlangwu** [comment]
> Also sorry, there are some merge conflicts which were introduced by this PR (if it helps to look at it
> #60)

I guess it is because I ran the `npm run lint:fix`, and it changed some format in some files. 
I have fixed it, can you please check is my PR  fine now? I have fixed the problem on the deploy preview too.

**Comment by trybick** [comment]
Awesome, thanks @tianlangwu !

**Pull request merged**